core/unit_test/TestGraph.hpp

```diff
@@ -866,6 +902,163 @@ TEST(TEST_CATEGORY, when_all_type) {
   static_assert(std::is_same_v<decltype(tail), node_ref_tail_t>);
 }
 
+#if defined(KOKKOS_ENABLE_CUDA) || defined(KOKKOS_ENABLE_HIP)
+template <GraphNodeType value, typename T, typename... Srcs>
+__global__ void set_to(T* const dst, const Srcs* const... srcs) {
+  dst[threadIdx.y] += (srcs[threadIdx.y] + ...) + static_cast<T>(value);
+}
+#endif
+
+template <typename Exec>
+struct ExternalCapture {
+  template <typename N, typename E, typename T, typename... Args>
+  static auto add(const N&, const E&, const T&, const T&, Args&&...) {
+    Kokkos::abort("Not implemented.");
+    return Kokkos::Experimental::GraphNodeRef<E>();
+  }
```

Comment:
Just wondering if we really need this version? Because we're doing `GTEST_SKIP` in the cases in which we wouldn't end up in the version below?